### PR TITLE
add constant $all_events

### PR DESCRIPTION
**Pull request by chzhcn**
for IRB

### CODE REVIEW

**Comment by ilyakamens** [comment]
LGTM except according to @tdumitrescu version bumps should be a separate commit

**Comment by chzhcn** [comment]
split the commit.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mixpanel-common",
   "main": "build/index.js",
-  "version": "0.3.5",
+  "version": "0.3.6",
```
Comment (tdumitrescu) [inline package.json]: minor PSA: in general, it's not a good practice to bump versions in pull requests. whoever publishes the new version will take care of that (in whatever order different branches end up getting merged)

**Comment by tdumitrescu** [comment]
like if https://github.com/mixpanel/mixpanel-common/pull/7 gets merged before this one, someone's just going to have to undo your version change anyway

File: package.json (hunk 1 above)
Comment (chzhcn) [inline package.json]: Makes sense. Removed that commit.

**Pull request merged**